Documentation/network/bgp-control-plane/bgp-control-plane.rst

```diff
@@ -15,6 +15,10 @@ from outside the cluster for environments that support BGP. Because BGP
 Control Plane does not program the :ref:`datapath <ebpf_datapath>`, do not use it to establish
 reachability within the cluster.
 
+.. admonition:: Video
+  :class: attention
+
+  For more insights on Cilium's BGP, check out `eCHO Episode 101: More BGP fun with Cilium <https://www.youtube.com/watch?v=Tv0R6VxyWhc>`__.
 .. _Border Gateway Protocol: https://datatracker.ietf.org/doc/html/rfc4271
```

Comment:
Please add an empty line here after the admonition.

Reply:
Also, s/Episode/episode/.

Reply:
Got it